gstreamerTypes.hpp

```diff
@@ -108,6 +108,131 @@ namespace gstreamer {
          */
         bool binary_in = false;
     };
+
+    /** Response from each connected peer to the RTP pipeline */
+    struct ReceivedResponse {
+        /** The SSRC of this source */
+        int rb_ssrc;
+        /** The SSRC of the sender of this RR */
+        int rb_sender_ssrc;
+        /** Lost 8-bit fraction */
+        int rb_fractionlost;
+        /** Lost packets */
+        int rb_packetslost;
+        /** Highest received seqnum */
+        int rb_exthighestseq;
+        /** Reception jitter (in clock rate units) */
+        int rb_jitter;
+        /** Last SR time (seconds in NTP Short Format, 16.16 fixed point) */
+        int rb_lsr;
+        /** Delay since last SR (seconds in NTP Short Format, 16.16 fixed point) */
+        int rb_dlsr;
+        /** The round-trip time (seconds in NTP Short Format, 16.16 fixed point) */
+        int rb_round_trip;
+    };
+
+    /** The RTP Stream Statistics
+     *
+     * This is the response when the stats is present on the pipeline
+*/
+    struct RTPStreamStatistics {
+        /** The label of the stream */
+        std::string label;
+
+        // The following fields are always present.
+        /** SSRC of this source */
+        int ssrc;
```

Comment:
Select bit size for ints. ... that is, use one of the `u?int[0-9]_t`

`ssrc` is 32 bit long, so `uint32_t`

Also, for big structures like this that will fill our logs a lot, try to "pack" things as much as possible, that is, try to put fields aligned on their size w.r.t. the struct beginning;

BAD, the compiler will add a uint8_t hole between field0 and field1, would be better to put field2 before field1

```
uint8_t field0;
uint16_t field1;
uint8_t field2;
```